PhysicsTools/PatUtils/interface/SmearedJetProducerT.h

```diff
@@ -298,5 +307,7 @@ class SmearedJetProducerT : public edm::stream::EDProducer<> {
         std::mt19937 m_random_generator;
 
         GreaterByPt<T> jetPtComparator;
+
+	int _nomVar;
```

Comment:
leading underscore is frowned upon.

Please change to ``` m_nomVar``` to be more in style with all other member data